README.md

```diff
@@ -1,193 +1,198 @@
 
 The instrumentation for the TNS app is as follows:
 
-- Metrics: Each tier of the TNS app exposes metrics on /metrics endpoints, which are scraped by the Grafana Agent. Additionally, these metrics are additionally tagged with exemplar information. The Grafana Agent then writes these metrics to a Prometheus (with remote-read enabled) for storage. [While the Prometheus could scrape metrics from the TNS App directly, the demo is configured to make the Agent the central point through which metrics, logs, and traces are collected. The Prometheus can be substituted for any backend which accepts Prometheus remote write, such as Thanos or Cortex.]
-- Logs: Each tier of the TNS app writes logs to standard output or standard error which is captured by Kubernetes, which are then collected by the Grafana Agent, which forwards them on to Loki for storage. 
+- Metrics: Each tier of the TNS app exposes metrics, tagged with [exemplar](https://grafana.com/docs/grafana/latest/basics/exemplars/) information, on `/metrics` endpoints. These are scraped by the Grafana Agent and then written to a Prometheus (with remote-read enabled) for storage. (While the Prometheus could scrape metrics from the TNS App directly, the demo is configured to make the Agent the central point through which metrics, logs, and traces are collected. The Prometheus can be substituted for any backend which accepts Prometheus remote write, such as Thanos or Cortex.)
 
-- Traces: Each tier of the TNS app sends traces in Jaeger format to the Grafana Agent, which then converts them to OTel format and forwards them to Tempo for storage. 
-Visualization: A Grafana instance configured to talk to the Prometheus, Loki, and Tempo instances makes it possible to query and visualize the metrics, logs, and traces data. 
+- Logs: Each tier of the TNS app writes logs to standard output or standard error. These logs are captured by Kubernetes and then collected by the Grafana Agent, which forwards them on to Loki for storage.
 
+- Traces: Each tier of the TNS app sends traces in Jaeger format to the Grafana Agent, which then converts them to OTel format and forwards them to Tempo for storage.
 
-## Prerequisites
+- Visualization: A Grafana instance configured to talk to the Prometheus, Loki, and Tempo instances makes it possible to query and visualize the metrics, logs, and traces data.
 
-Before you begin, install and configure the following software applications.
+## Prerequisites
 
 ### Docker
 
-Verify you have Docker installed. For download and installation instructions, click [here](https://docs.docker.com/install/).
+Verify you have Docker installed. Click [here](https://docs.docker.com/install/) for installation instructions.
 
-### K3D
+**Note:** Ensure that your Docker daemon has a minimum of 2.5 GB of total memory available for all pods in this demo to be scheduled.
```

Comment:
```suggestion
**Note:** Ensure that your Docker daemon has a minimum of 2.5 GB of total memory available for all pods in this demo application.
```

Reply:
Why the preference away from "scheduled"? Pod scheduling is what would be impacted by insufficient memory, so it seems prudent to mention here (it was also part of the original copy).